cmd/notary/keys.go

```diff
@@ -16,11 +16,12 @@ import (
 	"github.com/docker/notary/passphrase"
 	"github.com/docker/notary/trustmanager"
 
+	"io/ioutil"
```

Comment:
super nit (and not directly related to functionality): should this import just move right underneath `"io`"?